Updates for Rails 5

m5rk opened the pull request.

sandro committed:
Upgrade tilt dependency

sandro committed:
Upgrade sprocket engine to "Transform" style

sandro committed:
Don't escape attrs, use html_safe instead

sandro committed:
Fix tests

sandro committed:
Fix wrong asset paths by using Sprockets::Context

The image_tag and other url helpers were returning paths outside
of the asset pipeline, i.e. /images/icons/a.gif instead of
/assets/icons/a.gif

Commit:
Add .ruby-version

Commit:
Enable building / deploying on Google Cloud Build

Commit:
Bump version to 0.3.1

Commit:
Require rails 5.0

Commit:
Copy geminabox

Commit:
Actually run the test suite

Commit:
Generate Rails 5.2 dummy app

Commit:
Update dev dependencies

Commit:
Use expect syntax, fix stubbing

Commit:
The obligatory inception commit

mjroghelia commented:
Same comments as on https://github.com/carezone/obscured_ids/pull/3. The `app-test:latest` Docker tag is here too and looks like dev cruft. And I don't see how the SHA-tagged image gets pushed to the repo. The actual gem changes look ok.

domingusj committed:
Remove cache and publish image

The cache doesn't really get used that much, doesn't save much time, and adds some complexity. We also need to publish an image so it can be verified in the gem publishing step.

domingusj committed:
Fix typo

Commit:
Bump version to 0.3.2

Commit:
Bump

domingusj committed:
Limit files included in the build

We only need a few dirs and files to be included in the final gem, not
dotfiles, Dockerfiles, etc.

domingusj committed:
Bump version to 0.3.3
